lib/bamboo/phoenix.ex

```diff
@@ -98,22 +97,57 @@ defmodule Bamboo.Phoenix do
     raise "function implemented for documentation only, please call: use Bamboo.Phoenix"
   end
 
-
   @doc """
   Sets the layout when rendering HTML templates
+
+  ## Example
+
+      def html_email_layout do
+        new_email
+        |> from("myapp@example.com")
+        # Will use MyApp.LayoutView with email.html layout when rendering
+        |> put_html_layout({MyApp.LayoutView, "email.html"})
+      end
   """
   def put_html_layout(email, layout) do
     email |> put_private(:html_layout, layout)
   end
 
   @doc """
   Sets the layout when rendering plain text templates
+
+  ## Example
+
+      def text_email_layout do
+        new_email
+        |> from("myapp@example.com")
+        # Will use MyApp.LayoutView with email.text layout when rendering
+        |> put_text_layout({MyApp.LayoutView, "email.text"})
+      end
   """
   def put_text_layout(email, layout) do
     email |> put_private(:text_layout, layout)
   end
 
   @doc """
+  Sets the layout for rendering plain text and HTML templates
+
+  ## Example
+
+      def text_and_html_email_layout do
+        new_email
+        |> from("myapp@example.com")
```

Comment:
We could probably remove the `from` in these examples. Even though a from is required to send an email I think it's ok to leave it out for the example